templates/events/details.html

```diff
@@ -393,6 +418,55 @@ <h3>Meld deg av {{ event.title }} </h3>
                 </div>
             </div>
 
+            <!-- DIFF -->
+            <div id="attendees" class="modal fade">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                            <h3>Påmeldte til {{ event.title }} </h3>
+                            <h5><i>Siden dette er et sosialt arrangement kan du ønske å vise deg som påmeldt for andre å se.</i></h5>
+                        </div>
+                        <div class="modal-body">
+                            <table class="table table-striped table-condensed tablesorter attendees" id="extras-table">
+                                <thead>
+                                    <tr>
+                                        <th><b>Fornavn</b></th>
+                                        <th><b>Etternavn</b></th>
+                                    </tr>
+                                </thead>
+                                <tbody id="attendee_list">
+                                    {% for user in event.attendance_event.visible_attendees_qs %}
+                                    <tr>
+                                        <td>{{ user.first_name }}</td>
+                                        <td>{{ user.last_name }}</td>
+                                    </tr>
+                                    {% endfor %}
+                                </tbody>
+                            </table>
+                             {% if user_setting_show_as_attending %}
+                            <span><i>Du vises automatisk som påmeldt for alle sosiale arrangement, dette kan endres under personvern på <a href='/profile/privacy'>profilen din her.</a></i></span>
+                            {% else %}
+                            <span><i>Hvis du ønsker å automatisk vises som påmeldt på alle fremtidige sosiale arrangement kan det gjøres under personvern på <a href='/profile/privacy'>profilen din her.</a></i></span>
+                            {% endif %}
+
+                        </div>
+                        <div class="modal-footer">
+                            {% if user_attending %}
+                                {% if show_as_attending %}
+                                    <a href="{% url 'toggleShowAsAttending' event.id %}"><button class="btn btn-warning">Ikke vis meg på denne listen</button></a>
+                                {% else %}
+                                    <a href="{% url 'toggleShowAsAttending' event.id %}"><button class="btn btn-success">Gjør meg synlig som påmeldt</button></a>
+                                {% endif %}
+                            {% else %}
+                                <button class="btn btn-success disabled">Meld deg på arrangementet</button>
```

Comment:
Siden users som ikke er meldt på arrangementet ikke kan velge om de skal være synlige eller ikke, la jeg til en disabled knapp så det ikke bare var en lukk knapp for modulen. Er det nødvendig å ha en disabled knapp for users som ikke er meldt på arrangementet enda?

Reply:
Could you post a screenshot with what you mean?

I don't think the "Meld meg på"-button should be visible at all if you can not attend.

Reply:
I agree. I think the disabled *Meld deg på arrangementet* button creates confusion and should be removed.

<img width="1196" alt="disabled_button" src="https://user-images.githubusercontent.com/2287769/36389894-29f6535a-15a1-11e8-99c5-90af207a2402.png">